pkg/kubectl/resource_printer.go

```diff
@@ -476,10 +476,18 @@ func printPodTemplate(pod *api.PodTemplate, w io.Writer, withNamespace bool, wid
 
 	// Lay out all the other containers on separate lines.
 	for _, container := range containers {
-		_, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\n", "", container.Name, container.Image, "")
-		if err != nil {
-			return err
+		if withNamespace {
+			_, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", "", "", container.Name, container.Image, "")
```

Comment:
Minor, but you could put this in the if condition to make `err` local to the if statement.